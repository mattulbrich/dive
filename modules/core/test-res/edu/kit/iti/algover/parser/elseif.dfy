/* Was missing in the parser */

method ifthenelse(x : int) {

   if x > 10
   {
   }
   else if x > 5
   {
   }
   else if x > 0
   {
   }
   else
   {
   }

}
