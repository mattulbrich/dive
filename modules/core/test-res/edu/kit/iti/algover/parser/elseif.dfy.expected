(COMPILATION_UNIT
    (method ifthenelse (ARGS (VAR x (TYPE int)))
        (BLOCK (if (> x 10)
                BLOCK
                (BLOCK (if (> x 5)
                        BLOCK
                        (BLOCK (if (> x 0) BLOCK BLOCK))))))))
